src/MovieContainer/MovieContainer.css

```diff
@@ -3,4 +3,5 @@
     display: grid;
     grid-template-columns: repeat(3, 1fr);
     grid-gap: 20px;
-  }
\ No newline at end of file
+    height: 100vh;
```

Comment:
Looks like having 100vh is effecting the background. When I remove this styling value is goes back to normal

<img width="1415" alt="Screen Shot 2020-08-30 at 1 56 22 PM" src="https://user-images.githubusercontent.com/61163743/91668277-d53b1800-eac8-11ea-9c76-c615680c271e.png">

Reply:
Great catch!! This has been updated by adding a background on idex.css to `body`